test/system/010-images.bats

```diff
@@ -259,59 +259,25 @@ Labels.created_at | 20[0-9-]\\\+T[0-9:]\\\+Z
     run_podman inspect --format '{{.ID}}' $IMAGE
     imageID=$output
 
-    pauseImage=$(pause_image)
-    run_podman inspect --format '{{.ID}}' $pauseImage
-    pauseID=$output
+    run_podman pod inspect --format "{{.InfraContainerID}}" $pname
+    infra_ID="$output"
 
     run_podman 2 rmi -a
-    is "$output" "Error: 2 errors occurred:
-.** image used by .*: image is in use by a container: consider listing external containers and force-removing image
-.** image used by .*: image is in use by a container: consider listing external containers and force-removing image"
+    is "$output" "Error: image used by .*: image is in use by a container: consider listing external containers and force-removing image"
 
     run_podman rmi -af
     is "$output" "Untagged: $IMAGE
-Untagged: $pauseImage
-Deleted: $imageID
-Deleted: $pauseID" "infra images gets removed as well"
+Deleted: $imageID" "image gets removed"
 
     run_podman images --noheading
     is "$output" ""
-    run_podman ps --all --noheading
-    is "$output" ""
+    run_podman ps --all --noheading --no-trunc
+    assert "$output" =~ ".*$infra_ID.*" "infra container still running"
     run_podman pod ps --noheading
-    is "$output" ""
+    assert "$output" =~ ".*$pname.*" "pod still running"
 
-    run_podman create --pod new:$pname $IMAGE
     # Clean up
-    run_podman rm "${lines[-1]}"
     run_podman pod rm -a
-    run_podman rmi $pauseImage
-}
-
-# CANNOT BE PARALLELIZED: relies on exact output from podman images
-@test "podman images - rmi -f can remove infra images" {
-    pname=p_$(safename)
-    run_podman create --pod new:$pname $IMAGE
-
-    pauseImage=$(pause_image)
-    run_podman inspect --format '{{.ID}}' $pauseImage
-    pauseID=$output
```

Comment:
IMO we need to keep this, it was added as regression test for https://github.com/containers/podman/commit/12d762f8ee288164cdb1a8390520d88c4c5eb1bc

I think we should test this with an image still, something like this
```
podman pod create --infra-image quay.io/libpod/testimage:20241011 --infra-command "/home/podman/pause"  --name p1
```

Or likely just build your own image for this so we can delete it without deleting the main image, i.e. see
`podman pod create - hashtag AllTheOptions` test.